test/standalone/test-trace-header-context.js

```diff
@@ -16,18 +16,26 @@
 'use strict';
 
 var common = require('../hooks/common.js');
+var cls = require('../../src/cls.js');
 var http = require('http');
 var assert = require('assert');
-var express = require('../hooks/fixtures/express4');
 var constants = require('../../src/constants.js');
 
 describe('test-trace-header-context', function() {
-  beforeEach(function() {
-    require('../..').start();
+  var agent;
+  var express;
+  before(function() {
+    agent = require('../..')().startAgent({ samplingRate: 0 }).private_();
+    express = require('../hooks/fixtures/express4');
+  });
+
+  after(function() {
+    agent.stop();
   });
 
   afterEach(function() {
-    require('../..').stop();
+    cls.destroyNamespace();
```

Comment:
This change was needed for the tests to pass.  Matt helped me identify this fix.  I've added a TODO in the code to investigate this further (probably in a follow-up PR).